src/lib/code-sample/schema.ts

```diff
@@ -63,21 +71,36 @@ export const createCodeSample = (
         title: 'JavaScript',
         request: createJavascriptRequest(codeSampleDefinition, context),
         response: createJavascriptResponse(codeSampleDefinition, context),
+        request_syntax: 'javascript',
+        response_syntax: 'json',
```

Comment:
```suggestion
        response_syntax: 'javascript',
```

Reply:
Is it javascript, though? this one's a hard one 😖